### DISCUSSION TITLE
Argocd Upgrade currently running verion is v1.8.1

### DISCUSSION THREAD

**Discussion by rajeshgep** [argoproj/argo-cd#9910] [General]
Hi Team,

i want to upgrade argocd version v1.8.1 to newer versions which is more compaitable.

we are running EKS version 1.20 currently .

please guide me which argocd version is compaitable to suite kubernetes version 1.20 or 1.2x

Thanks,
Rajeshwar P.

**Comment by crenshaw-dev** [comment]
There's a PR currently open to add a support matrix. It's still waiting for a final review, but I think it is accurate: https://github.com/argoproj/argo-cd/pull/9876/files

I'd highly recommend upgrading to at least 2.2 if you can. It's the lowest version that is currently receiving security updates.
